pkgs/applications/kde/kmail.nix

```diff
@@ -96,15 +93,12 @@ mkDerivation {
     qtkeychain
     qtscript
     qtwebengine
-    akonadi-import-wizard
-    kaddressbook
-    kleopatra
-    pim-data-exporter
-  ];
+  ]
+  ++ pimExternalApplications.kmailApplications;
```

Comment:
```suggestion
  ] ++ pimExternalApplications.kmailApplications;
```

Reply:
I plan to run nixpkgs-fmt over this anyways, but sure I can include these changes already.